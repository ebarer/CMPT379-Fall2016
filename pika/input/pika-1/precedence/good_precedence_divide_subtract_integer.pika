exec {

  print 2 / 73 - 28, _n_.
  print 48 / 33 - 22, _n_.
  print 24 / 85 - 59, _n_.
  print 43 / 55 - 12, _n_.
  print 47 / 58 - 82, _n_.
  print 31 / 92 - 78, _n_.
  print 50 / 18 - 23, _n_.
  print 50 / 18 - 64, _n_.
  print 98 / 10 - 83, _n_.
  print 98 / 6 - 37, _n_.
  print 23 / 64 - 68, _n_.
  print 68 / 64 - 72, _n_.

  print 45 - 28 / 27, _n_.
  print 99 - 40 / 94, _n_.
  print 20 - 30 / 51, _n_.
  print 5 - 97 / 68, _n_.
  print 38 - 53 / 57, _n_.
  print 33 - 91 / 70, _n_.
  print 26 - 64 / 97, _n_.
  print 1 - 99 / 28, _n_.
  print 32 - 57 / 12, _n_.
  print 81 - 19 / 43, _n_.
  print 15 - 59 / 32, _n_.
  print 80 - 37 / 93, _n_.

  print 26 / 4 - 98 / 28, _n_.
  print 99 / 61 - 99 / 19, _n_.
  print 25 / 17 - 63 / 29, _n_.
  print 68 / 90 - 60 / 12, _n_.
  print 22 / 17 - 77 / 64, _n_.
  print 2 / 18 - 94 / 34, _n_.
  print 87 / 13 - 8 / 6, _n_.
  print 81 / 31 - 18 / 30, _n_.
  print 31 / 68 - 55 / 26, _n_.
  print 1 / 65 - 73 / 51, _n_.
  print 15 / 42 - 23 / 4, _n_.
  print 52 / 9 - 66 / 68, _n_.

  print 76 - 42 / 64 - 76, _n_.
  print 34 - 63 / 38 - 91, _n_.
  print 21 - 97 / 14 - 32, _n_.
  print 72 - 91 / 80 - 37, _n_.
  print 65 - 41 / 94 - 48, _n_.
  print 7 - 37 / 32 - 44, _n_.
  print 17 - 65 / 49 - 97, _n_.
  print 97 - 80 / 11 - 40, _n_.
  print 35 - 30 / 83 - 13, _n_.
  print 41 - 46 / 55 - 53, _n_.
  print 23 - 58 / 50 - 49, _n_.
  print 17 - 74 / 47 - 42, _n_.

}

