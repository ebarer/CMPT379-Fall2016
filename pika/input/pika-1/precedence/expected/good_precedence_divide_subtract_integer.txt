-28
-21
-59
-12
-82
-78
-21
-62
-74
-21
-68
-71
44
99
20
4
38
32
26
-2
28
81
14
80
3
-4
-1
-5
0
-2
5
2
-2
-1
-5
5
0
-58
-17
34
17
-38
-81
50
22
-12
-27
-26
